website/fiddle/public/wc/luigiExampleWC.js

```diff
@@ -4,36 +4,12 @@ import "./f_input.js";
 
 const template = document.createElement('template');
 template.innerHTML = `
+      <style>
+        .fd-page{
+          padding: 20px;
+        }
+      </style>
 
-<!DOCTYPE html>
-<html lang="EN">
-<head>
-    <meta charset="utf-8">
-    <title></title>
-    <link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.css" rel="stylesheet">
-    
-    <style>
-      .fd-page{
-        padding: 20px;
-      }
-    </style>
-    <script>
-        function toggleElAttrs(id, toggleAttrs) {
-            let ref = document.getElementById(id);
-            if (ref && Array.isArray(toggleAttrs) && toggleAttrs.length){
-                for(var i = 0; i < toggleAttrs.length; i++) {
-                    var val = ref.getAttribute(toggleAttrs[i]);
-                    if(val === 'true') {
-                        setElAttr(id, toggleAttrs[i], 'false');
-                    } else if (val === 'false') {
-                        setElAttr(id, toggleAttrs[i], 'true');
-                    }
-                }
-            }
-       }
-    </script>
```

Comment:
Do we need to remove the script as well ?

Reply:
Hmm, I looked at it and have found that it has no uses, I see now that it exists as well in ```docs\backup\web-component-to-be-used-in-expert-page.md``` as a part of an example. I guess it could stay or the example should be updated accordingly to what I have already removed.

Which one should it be, do you know? :)

Reply:
I think we'd rather keep this. It might be used in another example somewhere I would suppose.